Description: Adds the name of the distribution to the version of Tomcat
 reported on the error pages (i.e. 'Apache Tomcat/8.0.x (Debian)')
Author: Yolanda Robla <yolanda.robla@canonical.com>
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/729840
--- a/java/org/apache/catalina/util/ServerInfo.properties
+++ b/java/org/apache/catalina/util/ServerInfo.properties
@@ -13,6 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-server.info=Apache Tomcat/@VERSION@
+server.info=Apache Tomcat/@VERSION@ (@TOMCAT_DISTRIBUTION@)
 server.number=@VERSION_NUMBER@
 server.built=@VERSION_BUILT@
\ No newline at end of file
--- a/build.xml
+++ b/build.xml
@@ -266,6 +266,7 @@
     <filter token="YEAR" value="${year}"/>
     <filter token="VERSION" value="${version}"/>
     <filter token="VERSION_NUMBER" value="${version.number}"/>
+    <filter token="TOMCAT_DISTRIBUTION" value="${distribution.name}"/>
     <filter token="VERSION_MAJOR" value="${version.major}"/>
     <filter token="VERSION_MAJOR_MINOR" value="${version.major.minor}"/>
     <filter token="VERSION_BUILT" value="${today} ${tstamp} UTC"/>
